### PR TITLE
fix: backport for AUT-2716

### PR DESCRIPTION
**Pull request by Silvia-loza**
Backport to nov release for AUT-2716 [https://oat-sa.atlassian.net/browse/AUT-2716](https://github.com/oat-sa/tao-community/pull/url)

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -11647,5 +11647,5 @@
     "platform-overrides": {
         "php": "7.4"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.1.0"
```
Comment (marpesia) [inline composer.lock:11650]: is this ok?

**Review by bartlomiejmarszal — APPROVED** [review]

**Review by bziondik — APPROVED** [review]
LGTM

File: composer.lock (hunk 1 above)
Comment (Silvia-loza) [reply composer.lock:11650]: is the result of updating version of `oat-sa/extension-tao-itemqti` in composer to `29.14.4.3` and running `composer update oat-sa/extension-tao-itemqti` 
Not sure why the version decreased to be honest...  image

Comment (marpesia) [reply composer.lock:11650]: Maybe it is ok. Look at the previous PR https://github.com/oat-sa/tao-community/commit/7b64b2b658c18e8b808e14635b665f4f534db45a

**Review by marpesia — APPROVED** [review]

**Pull request merged**